package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -662,10 +659,13 @@ def _read_frame(self, frame):
             ts._forces[:] = (self.trjfile.variables['forces'][frame] *
                              self.scale_factors['forces'])
         if self.periodic:
-            ts._unitcell[:3] = (self.trjfile.variables['cell_lengths'][frame] *
-                                self.scale_factors['cell_lengths'])
-            ts._unitcell[3:] = (self.trjfile.variables['cell_angles'][frame] *
-                                self.scale_factors['cell_angles'])
+            unitcell = np.zeros(6)
```

Comment:
maybe this should be obvious, but did you use this instead of `np.empty()` because of a preference for having `0` as a placeholder in the absence of the appropriate unit cell info?

It is such a small array that the performance difference is likely irrelevant of course.

Reply:
Yeah internally (`ts._unitcell`) we're still using zeros as the way to indicate no box